### PR TITLE
polypolish 0.6.0

### PR DESCRIPTION
**Pull request by erinyoung**
Polypolish has had a recent breaking update.

Notably, the removal of polypolish_insert_filter.py, which changes how this tool is used.

I copied the [prior Dockerfile](https://github.com/StaPH-B/docker-builds/blob/master/polypolish/0.5.0/Dockerfile), but I've made some substantial changes:
- there is now a builder stage, which uses rust as its base
  - polypolish is now built from source and the copied to the app stage
- the test had to change to accommodate the removal of polypolish_insert_filter.py
- added CMD layer

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [X] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: polypolish/0.6.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    procps \
+    unzip \
+    python3 && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /usr/local/bin
```
Comment (Kincekara) [inline polypolish/0.6.0/Dockerfile:36]: I think this is redundant.

File: polypolish/0.6.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+RUN wget -q https://github.com/rrwick/Polypolish/archive/refs/tags/v${POLYPOLISH_VER}.tar.gz && \
+    tar -vxf v${POLYPOLISH_VER}.tar.gz && \
+    cd /Polypolish-${POLYPOLISH_VER} && \
+    cargo build --release 
+
+FROM ubuntu:jammy as app
+
+ARG POLYPOLISH_VER
+
+COPY --from=builder /Polypolish-${POLYPOLISH_VER}/target/release/polypolish /usr/local/bin/polypolish
```
Comment (Kincekara) [inline polypolish/0.6.0/Dockerfile:16]: It is better to have the COPY line after installing dependencies to maintain the same style as other Dockerfiles.

Comment (Kincekara) [reply polypolish/0.6.0/Dockerfile:16]: Just asking out of curiosity; is the prebuilt binary not working on Jammy?

**Comment by Kincekara** [comment]
Just asking out of curiosity; is the prebuilt binary not working on Jammy?

**Comment by erinyoung** [comment]
I initially tried the prebuilt binary and ran into issues. Unfortunately I didn't keep any notes and I don't remember what those issues were.

**Comment by Kincekara** [comment]
Thank you for your reply and changes. This looks great. I am merging it.

**Review by Kincekara — APPROVED** [review]

**Comment by Kincekara** [comment]
Thank you for putting altogether this!
You can check the image deployment from here: https://github.com/StaPH-B/docker-builds/actions/runs/7995283808.

**Pull request merged**